src/extension.ts

```diff
@@ -246,6 +246,8 @@ function initCommandAsync(context: vscode.ExtensionContext,
         let errorData: ErrorData | undefined;
         const properties: { [key: string]: string; } = {};
         properties.result = "Succeeded";
+        properties.fromCommandPalette = (!args || !args[0]).toString();
```

Comment:
Can we make sure all the context menu triggered has args[0] not null?

Reply:
Yes, args[0] for all commands not triggered from command palette is not null